SPECS/net-snmp/net-snmp.spec

```diff
@@ -96,6 +97,8 @@ rm -rf %{buildroot}/*
 %exclude /usr/lib/perl5/perllocal.pod
 
 %changelog
+*   Thu Feb 04 2021 Henry Li <lihl@microsoft.com> 5.8-6
```

Comment:
Please also run the `spec-cleaner` on this SPEC, the changelog should be in a slightly different format.

Reply:
Fixed